docker-compose.yml

```diff
@@ -1,32 +1,36 @@
-version: '3'
+version: '3.7'
+
 services:
+
     postgres:
         build: ./docker/postgres/
         environment:
             POSTGRES_USER: linkedevents
             POSTGRES_PASSWORD: linkedevents
             POSTGRES_DB: linkedevents
         ports:
-            - "5432:5432"
+            - "127.0.0.1:5555:5432"
         volumes:
             - postgres-data-volume:/var/lib/postgres/data
-        container_name: linkedevents-db
 
     django:
         build:
             context: ./
             dockerfile: ./docker/django/Dockerfile
-        command: python manage.py runserver 0.0.0.0:8000
+            target: development
         environment:
-            DATABASE_URL: postgres://linkedevents:linkedevents@linkedevents-db/linkedevents
+            - APPLY_MIGRATIONS=true
+            - CREATE_SUPERUSER=true
+            - DEBUG=true
+            - DATABASE_URL=postgis://linkedevents:linkedevents@postgres/linkedevents
+            - DEV_SERVER=true
+            - RUNSERVER_ADDRESS=0.0.0.0:8000
+            - WAIT_FOR_IT_ADDRESS=postgres:5432
         volumes:
-            - .:/code
+            - .:/app:cached
             - django-media-volume:/var/media/
         ports:
-            - "8000:8000"
-        depends_on:
-            - postgres
-        container_name: linkedevents-backend
+            - "127.0.0.1:8000:8000"
```

Comment:
this is a more secure option, yes, but it might cause issues, if/when you'll try to connect to this container via a docker network. e.g. if you will have several containerized City-of-Helsinki projects trying to talk to each other on `helsinki` network. then this container will not be accessible for the others.

Reply:
hmm.. I've been under the assumption that when using the [short syntax](https://docs.docker.com/compose/compose-file/#ports) for ports in the form of `HOST:CONTAINER`, what's on the left only matters to the host machine and containers can talk with each other in their own "Docker" network (I don't know the correct term to refer to this "Docker" network).. and so in this case, a frontend container running on port 3000 can talk with this linkedevents backend on port 8000 because of the right side of the short syntax. 

Is that incorrect?

Reply:
ah, yes, you are right. checked it just now and docker containers use the port on the right side, no matter if it is an explicit network (like the `helsinki` one we have in the newer dockerized projects) or an implicit one (like the one docker creates for the containers from a `docker-compose.yml`). 👍

Reply:
Cool. I also updated the default network name to `helsinki` in d8146d71a7bee7cdcd1440e53456cdbfb25610a6 for uniformness among our newly Dockerized projects.

Reply:
that is a welcome addition 👍